redhat-developer/rhdh-operator
# PVCs default config

## Description

Adds ability to declare PVC(s) in the default config to be mounted to the Backstage container.

## Which issue(s) does this PR fix or relate to

https://issues.redhat.com/browse/RHIDP-3622

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
- Add pvcs.yaml to the default-configuration according to configuration.md doc. Also see integration_tests/pvcs_test.go
- Check if corresponfing PVC(s) added and mounted to /opt/app-root/src/backstage-<cr-name>-<pvc-name> folders
